Source/controls/plrctrls.cpp

```diff
@@ -11,10 +11,7 @@
 #include "automap.h"
 #include "control.h"
 #include "controls/controller_motion.h"
-#include "miniwin/misc_msg.h"
-#ifndef USE_SDL1
 #include "controls/devices/game_controller.h"

```

Comment:
Removing this check is probably going to break SDL1
```suggestion
#ifndef USE_SDL1
#include "controls/devices/game_controller.h"
#endif
```

Reply:
I thought I needed game_controller.h regardless of SDL version